compliance_checker/tests/test_feature_detection.py

```diff
@@ -226,6 +226,20 @@ def test_auxiliary_coordinates(self):
             aux_coord_vards = util.get_auxiliary_coordinate_variables(nc)
             assert set(['lat', 'lon']) == set(aux_coord_vards)
 
+    def test_forecast_reference_metadata(self):
+        '''
+        Tests variables used for forecast reference metadata to ensure they are
+        not misclassified as geophysical variables.
+        '''
+        with Dataset(resources.STATIC_FILES['forecast_reference']) as nc:
+            assert util.is_geophysical(nc, 'forecast_reference_time') is False
```

Comment:
Use `self.assertTrue`, `self.assertFalse`, and `self.assertEqual` in the below methods.  In the event the tests fail, the error messages are more useful.  `assert` can also be ignored completely in Python using the `-O` flag.  While I haven't ever seen the aforementioned flag used in practice, I still prefer using `unittest` methods for most unit test assertion.

Reply:
:heavy_check_mark: